### PR TITLE
Fixed test pipeline

### PR DESCRIPTION
**Pull request by shibshib**
This PR makes our test pipeline green again.

### CODE REVIEW

File: bittensor/_subtensor/subtensor_impl.py [hunk 1]
```diff
@@ -661,7 +661,8 @@ def unstake (
         if response.is_success:
             with bittensor.__console__.status(":satellite: Checking Balance on: ([white]{}[/white] ...".format(self.network)):
                 new_balance = self.get_balance( wallet.coldkey.ss58_address )
-                new_stake = bittensor.Balance.from_tao( self.neuron_for_uid( uid = neuron.uid, ss58_hotkey = wallet.hotkey.ss58_address ).stake)
+                block = self.get_current_block()
```
Comment (unconst) [inline bittensor/_subtensor/subtensor_impl.py:664]: This should not need to happen

File: tests/unit_tests/bittensor_tests/test_axon.py [hunk 2]
```diff
@@ -20,9 +20,9 @@
 axon = bittensor.axon(wallet = wallet)
 
 def sign(wallet):
-    nounce = int(time.time() * 1000)
+    nounce = str(int(time.time() * 1000))
     receptor_uid = str(uuid.uuid1())
-    message  = nounce + str(wallet.hotkey.ss58_address) + receptor_uid
+    message  = "{}{}{}".format(nounce, str(wallet.hotkey.ss58_address), receptor_uid)
```
Comment (unconst) [inline tests/unit_tests/bittensor_tests/test_axon.py:25]: Nice

File: tests/unit_tests/bittensor_tests/test_subtensor.py [hunk 3]
```diff
@@ -16,14 +16,14 @@ def setUp(self):
         self.wallet.set_hotkey(Keypair.create_from_mnemonic(Keypair.generate_mnemonic()), encrypt=False, overwrite=True)
         self.mock_neuron = self.subtensor._neuron_dict_to_namespace(
             dict({
-                "version":0,
+                "version":1,
```
Comment (unconst) [inline tests/unit_tests/bittensor_tests/test_subtensor.py:19]: Why was this required?

**Review by unconst — APPROVED** [review]

File: tests/unit_tests/bittensor_tests/test_subtensor.py (hunk 3 above)
Comment (shibshib) [reply tests/unit_tests/bittensor_tests/test_subtensor.py:19]: There was a check in neuron dict to namespace that if version is 0 then return a null neuron

File: bittensor/_subtensor/subtensor_impl.py (hunk 1 above)
Comment (shibshib) [reply bittensor/_subtensor/subtensor_impl.py:664]: yeah but doesn't work without it? maybe i was out of date?
